manifest.json

```diff
@@ -1240,6 +1240,20 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "architecture": "arm",
+              "dockerfile": "src/raspbian/10/helix/arm32v6",
+              "os": "linux",
+              "osVersion": "buster",
+              "tags": {
+                "raspbian-10-helix-arm32v6-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {}
```

Comment:
Does this run on arm32v7/arm64v8 stuff or does it require matching?  If the latter, I don't think we have devices.

Reply:
It'll run on those, yeah, although at a performance penalty (n^cores). We do have a queue - Raspbian.9.ARM32.IoT.Open

Reply:
(https://apebox.org/wordpress/linux/1281 is my blog post on the topic)